Lib/test/test_pdb.py

```diff
@@ -2587,7 +2587,7 @@ def quux():
     pass
 """.encode(),
             'bœr',
-            ('bœr', 4),
+            ('bœr', 5),
```

Comment:
Why did this change?

Reply:
As mentioned in the other comment, this used to locate the line of `def bœr():`, now it's `pass`, so one line change